### PR TITLE
Phase2-gex85 Update cfg's in Validation/Geometry/test in view of changes for SD's

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Update cfg's in Validation/Geometry/test in view of changes for SD's

#### PR validation:

Tested the cfg's against failures

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Validation/Geometry/test/runP_Mtd_cfg.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Phase2_cff import Phase2
 
-process = cms.Process("PROD")
+process = cms.Process("PROD",Phase2)
```
Comment (fabiocos) [inline Validation/Geometry/test/runP_Mtd_cfg.py:4]: @bsunanda is this the most appropriate modifier for this geometry scenario?

Comment (bsunanda) [reply Validation/Geometry/test/runP_Mtd_cfg.py:4]: This qualifier will work for all MTD configurations. The real issue is the choice of SD which is true for all Phase2 scenarios

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by jfernan2** [comment]
+1

**Comment by jfernan2** [comment]
@bsunanda @civanch let me take this opportunity to kindly request from you to perform the esConsumes migration in this package as mentioned in:
https://github.com/cms-sw/cmssw/issues/31061
Thanks

**Comment by qliphy** [comment]
possible fix for https://github.com/cms-sw/cmssw/issues/34601

**Comment by qliphy** [comment]
+1

**Pull request merged**